projects/kit/components/input-password/input-password.options.ts

```diff
@@ -13,9 +13,9 @@ export interface TuiInputPasswordOptions {
 /** Default values for the input password options. */
 export const TUI_INPUT_PASSWORD_DEFAULT_OPTIONS: TuiInputPasswordOptions = {
     icons: {
-        hide: ({$implicit}) =>
+        hide: ({$implicit}) => ($implicit === `s` ? `tuiIconEye` : `tuiIconEyeLarge`),
+        show: ({$implicit}) =>
```

Comment:
Is it breaking change ?

Reply:
How about we do this in 4.0 and override only proprietary icons for now?